src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlQueueClient.cs

```diff
@@ -360,5 +300,29 @@ public async Task<bool> PutJobHeartbeatAsync(JobInfo jobInfo, CancellationToken
 
             return cancel;
         }
+
+        // Class copied from src\Microsoft.Health.Fhir.SqlServer\Features\Schema\Model\VLatest.Generated.cs .
```

Comment:
Can you add the comment why we did this?

Reply:
I'll add comment to the source file. This comes from the autogenerated file which uses SqlCcommandWrapper as an input param. We are not using SqlCcommandWrapper any more, we are using SqlCommand. Sergey suggested to just copy autogenerated code here and modify instead of looking how to change autogenerated code.